server/gfmd/db_pgsql.c

```diff
@@ -4255,6 +4255,141 @@ gfarm_pgsql_mdhost_load(void *closure,
 	return (GFARM_ERR_NO_ERROR);
 }
 
+
+/**********************************************************************/
+
+static gfarm_error_t
+pgsql_user_auth_update(gfarm_uint64_t seqnum, struct db_user_auth_arg *info,
+	const char *sql,
+	gfarm_error_t (*check)(PGresult *, const char *, const char *),
+	const char *diag)
+{
+	PGresult *res;
+	const char *paramValues[3];
+	gfarm_error_t e;
+
+	if ((e = gfarm_pgsql_start(diag)) != GFARM_ERR_NO_ERROR) {
+		gflog_debug(GFARM_MSG_1003247,
```

Comment:
change GFARM_MSG_1003247 to GFARM_MSG_UNFIXED

Reply:
commit [429eb3d](https://github.com/oss-tsukuba/gfarm/pull/1141/commits/429eb3daa151165a09117234d565149a2a1ac4e6) has not applied above change.
Could you change it?